request/ta_find_transaction_objects.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2018-2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "ta_find_transaction_objects.h"
+
+ta_find_transaction_objects_req_t* ta_find_transaction_objects_req_new() {
+  ta_find_transaction_objects_req_t* req =
+      (ta_find_transaction_objects_req_t*)malloc(sizeof(ta_find_transaction_objects_req_t));
+  if (req != NULL) {
+    req->hashes = NULL;
+    return req;
+  }
+  return NULL;
+}
+
+status_t ta_find_transaction_objects_req_push_address(ta_find_transaction_objects_req_t* req,
```

Comment:
I don't think this kind of setter is needed if there's only one operation. It takes extra step for me knowing what it actually does.